test-data/snake_oil_field/snake_oil_surface.ert

```diff
@@ -9,6 +9,7 @@ RANDOM_SEED 3593114179000630026631423308983283277868
 
 RUNPATH <STORAGE>/runpath/realization-<IENS>/iter-<ITER>
 ENSPATH <STORAGE>/ensemble
+JOBNAME SNAKE_OIL_<IENS>
```

Comment:
Is this needed then?

Reply:
It's not necessarily needed, but it is recommended by the docs so I thought I would add it. Without this, running on the queue creates a bunch of jobs with the same name.